charts/consul/templates/server-statefulset.yaml

```diff
@@ -254,42 +255,54 @@ spec:
                   name: {{ .Values.global.acls.replicationToken.secretName | quote }}
                   key: {{ .Values.global.acls.replicationToken.secretKey | quote }}
             {{- end }}
-            {{- if and .Values.global.cloud.enabled .Values.global.cloud.secretName }}
+            {{- if and .Values.global.cloud.enabled}}
```

Comment:
changed so the check for the presence of a value is on each individual secret below.

Reply:
Can we cut the `and` then?